pkg/vm/emit/emit.go

```diff
@@ -119,6 +119,8 @@ func Array(w *io.BinWriter, es ...any) {
 			Array(w, e...)
 		case int64:
 			Int(w, e)
+		case uint64:
+			Int(w, int64(e))
```

Comment:
Have you tried pushing `math.MaxUint64` this way?

Reply:
Oh, my bad, I kept in mind that they have the same size and conversion will be successful, but the result will be far from the user's expectations.